buildenv/jenkins/openjdk_tests

```diff
@@ -1,121 +1,145 @@
 #!groovy
 
-def PLATFORM_MAP = [ 
+def PLATFORM_MAP = [
     'arm_linux' : [
         'SPEC' : 'linux_arm',
-        'LABEL' : 'ci.role.test&&sw.os.linux&&hw.arch.aarch32',
+        'LABEL' : 'ci.role.test&&sw.os.linux&&hw.arch.aarch32'
```

Comment:
I would like to keep the trailing comma. This was added purposely 
- if there is a new entry, this line will not change
- it prevents the error in future

Reply:
Leaving trailing commas simplifies the git history. If we merge this PR, all of those lines will be updated, and when we look for one line history, we will find this PR and not the reason for the line in the first place.
Please see the article https://time2hack.com/are-you-using-trailing-commas-in-your-javascript/
The git commits are much cleaner if we add an extra comma.
I understand that this is groovy code and not JavaScript, but the same logic applies.